src/backend/marsha/core/utils/react_locales_utils.py

```diff
@@ -0,0 +1,19 @@
+"""Helper matching a django locale to a react locale."""
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.translation import to_locale
+
+
+def react_locale(value):
+    """Convert a language (simple ISO639-1 or full language with regions) to an ISO15897 locale."""
```

Comment:
it seems to me that this should be called `language_to_locale`

I'm uneasy seeing stuff with "react" names in the backend as this brings bad memories. `react` is also an implementation detail of the frontend side that should not appear anywhere in the backend in my opinion.

WDYT?

Reply:
but in this case it is not just a language_to_locale shift. It is a mapping between Django's language and React's language. Call it backend and frontend maybe?

Reply:
I mean the function docstring is `Convert a language (simple ISO639-1 or full language with regions) to an ISO15897 locale.`

Is that not language_to_locale pretty much?

Reply:
In fact it should be to a locale `supported by React`.

Reply:
@sampaccoud thanks for your explanation: `language_to_locale_preferred` maybe then?

Reply:
I updated the docstring with richie's description. It is mentionned that this locale is supported by react.